pallets/eth-bridge/src/tests/cancel.rs

```diff
@@ -47,7 +47,8 @@ use crate::tests::{
 };
 use crate::{AssetConfig, EthAddress};
 use common::{
-    AssetInfoProvider, AssetName, AssetSymbol, DEFAULT_BALANCE_PRECISION, DOT, KSM, USDT, VAL, XOR,
+    AssetInfoProvider, AssetName, AssetSymbol, DEFAULT_BALANCE_PRECISION, DOT, KSM, PSWAP, USDT,
+    VAL, XOR,
```

Comment:
Why do you add new assets in `use`? And in other files.
I don't see they are used here.

Reply:
In the pallet it was implemented as constant, which duplicate common const, so I removed it and import. And maybe somewhere it used manually (path::const) and I changed to import. Anyway it is auto checked and unused import won't exist.